corehq/apps/app_manager/static/app_manager/js/app_manager.js

```diff
@@ -247,7 +248,34 @@ hqDefine('app_manager/js/app_manager', function () {
                 $(this).closest('.js-sorted-li').removeClass('appnav-highlight');
             });
         }
-
+        function nestChildModules() {
+            var modulesByUid = {},
+                childModules = [];
+            $(".module").each(function (index, element) {
+                $(element).data('index', index);
+                modulesByUid[ $(element).data('uid') ] = element;
+                if ( $(element).data('rootmoduleid') ) {
```

Comment:
Should this be `rootmoduleuid`? I just went back to check this wouldn't run into issues with rootmoduleid being zero, rootmoduleuid would make it more obvious that isn't an issue.